src/command.cc

```diff
@@ -68,18 +68,27 @@ ProbeSSBOCommand* Command::AsProbeSSBO() {
   return static_cast<ProbeSSBOCommand*>(this);
 }
 
-DrawRectCommand::DrawRectCommand(PipelineData data)
-    : Command(Type::kDrawRect), data_(data) {}
+DrawRectCommand::DrawRectCommand(
+    PipelineData data,
+    bool is_pipeline_data_different_from_previous_one)
```

Comment:
I don't think this bool should be part of the commands as it doesn't mean anything to the command. It should be up to the backend to figure this out automatically.